generative_ai/evaluation/pairwise_summarization_quality.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from vertexai.preview.evaluation import EvalResult
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def evaluate_output() -> EvalResult:
+    # [START generativeaionvertexai_evaluate_output_pairwise_summarization_quality]
+    import pandas as pd
+
+    import vertexai
+    from vertexai.generative_models import GenerativeModel
+    from vertexai.evaluation import (
+        EvalTask,
+        PairwiseMetric,
+        MetricPromptTemplateExamples,
+    )
+
+    # TODO(developer): Update & uncomment line below
+    # PROJECT_ID = "your-project-id"
+    vertexai.init(project=PROJECT_ID, location="us-central1")
+
+    prompt = """
+    Summarize the text such that a five-year-old can understand.
+
+    # Text
+
+    As part of a comprehensive initiative to tackle urban congestion and foster
+    sustainable urban living, a major city has revealed ambitious plans for an
+    extensive overhaul of its public transportation system. The project aims not
+    only to improve the efficiency and reliability of public transit but also to
+    reduce the city\'s carbon footprint and promote eco-friendly commuting options.
+    City officials anticipate that this strategic investment will enhance
+    accessibility for residents and visitors alike, ushering in a new era of
+    efficient, environmentally conscious urban transportation.
+    """
+
+    eval_dataset = pd.DataFrame({"prompt": [prompt]})
+
+    # Baseline model for pairwise comparison
+    baseline_model = GenerativeModel("gemini-1.5-pro")
```

Comment:
Can you please test and use `gemini-1.5-pro-002` or `gemini-1.5-pro-001` ?

Recommendation is to include 001 or 002 (latest version id's)

Reply:
done